### PR TITLE
{lang}[dummy/dummy] Java v1.8.0_141

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/j/Java/Java-1.8.0_141.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name = 'Java'
+version = '1.8.0_141'
```
Comment (verdurin) [inline easybuild/easyconfigs/j/Java/Java-1.8.0_141.eb]: @JackPerdue worth updating to `1.8.0_144`, or did you need this specific version?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
ca004.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d7edba77a0a52bbfc5289e8cbb4dc2a7 for a full test report.

File: easybuild/easyconfigs/j/Java/Java-1.8.0_141.eb (hunk 1 above)
Comment (JackPerdue) [inline easybuild/easyconfigs/j/Java/Java-1.8.0_141.eb]: ugh..  it just feels so wrong to have to go to Oracle for Java..... but I'll do it.

Java is a mess given EasyBuild's need for reproducibility.   Calling it just 1.8 that gets updated would be nice.   Or using Lmod's ranges (x>=1.8 && x<1.9) for modules so old versions could be purged.

I'll submit a new PR after I dirty myself again at Oracle.

Comment (JackPerdue) [inline easybuild/easyconfigs/j/Java/Java-1.8.0_141.eb]: p.s. I'll submit a new CPLEX too

Comment (JackPerdue) [inline easybuild/easyconfigs/j/Java/Java-1.8.0_141.eb]: BTW... when I submitted this, 141 was current.   I can't help EB turnaround time  on PRs let another update come out. :)     144 shortly

**Comment by JackPerdue** [comment]
Now PR #4968

File: easybuild/easyconfigs/j/Java/Java-1.8.0_141.eb (hunk 1 above)
Comment (verdurin) [inline easybuild/easyconfigs/j/Java/Java-1.8.0_141.eb]: @JackPerdue I'm doing my best to keep up with your barrage...

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
ca005.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a459701ca37f2a6b39e87ee745da62c4 for a full test report.

**Comment by verdurin** [comment]
Merging, thanks @JackPerdue

**Pull request merged**